tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -573,4 +574,99 @@ spec:
 			helper.CmdShouldPass("odo", "service", "delete", "EtcdCluster/etcd", "-f")
 		})
 	})
+
+	Context("Operators are installed in the cluster", func() {
+
+		JustBeforeEach(func() {
+			// wait till odo can see that all operators installed by setup script in the namespace
+			odoArgs := []string{"catalog", "list", "services"}
+			operators := []string{"etcdoperator", "service-binding-operator"}
+			for _, operator := range operators {
+				helper.WaitForCmdOut("odo", odoArgs, 5, true, func(output string) bool {
+					return strings.Contains(output, operator)
+				})
+			}
+		})
+
+		JustAfterEach(func() {
+			helper.DeleteProject(commonVar.Project)
+		})
```

Comment:
We can use the `preSetup()` and `cleanPreSetup()` here
```
JustBeforeEach(func() {
	preSetup()
})

JustAfterEach(func() {
	cleanPreSetup()
})
```

Reply:
These functions have been removed in the latest master

Reply:
With the new approcah of tests, the preSetup() and cleanPreSetup() is called only once in the most outer `When`, we could inline the code instead, to make the flow more readable